docs/source/installation/guide.rst

```diff
@@ -44,25 +44,30 @@ MVAPICH (for details, see `the official instructions <http://mvapich.cse.ohio-st
 
 .. _nccl-install:
   
-NVIDIA NCCL
-~~~~~~~~~~~
+NCCL
+~~~~
 
 To enable efficient intra-node GPU-to-GPU communication,
-we use `NVIDIA NCCL <https://github.com/NVIDIA/nccl>`_.
-See `the official instructions <https://github.com/NVIDIA/nccl#build--run>`_ for installation.
-
-Please properly configure environment variables to expose NCCL both when you install and use ChainerMN.
-Typical configurations should look like the following::
-
-  export NCCL_ROOT=<path to NCCL directory>
-  export CPATH=$NCCL_ROOT/include:$CPATH
-  export LD_LIBRARY_PATH=$NCCL_ROOT/lib/:$LD_LIBRARY_PATH
-  export LIBRARY_PATH=$NCCL_ROOT/lib/:$LIBRARY_PATH
+we use `NVIDIA Collective Communications Library (NCCL) <https://developer.nvidia.com/nccl>`_.
+See `the official instructions <http://docs.nvidia.com/deeplearning/sdk/nccl-developer-guide/index.html#downloadnccl>`_ for installation.
 
 ChainerMN requires NCCL even if you have only one GPU per node.  The
 only exception is when you run ChainerMN on CPU-only environments. See
 :ref:`non-gpu-env` for more details.
 
+.. note::
+
+   We reccomend NCCL 2.0+ but NCCL 1.0 can be used.
```

Comment:
`NCCL 1.0` We actually do not even have access to NCCL 1.0. NCCL 1.2.X and 1.3.X are only available online. https://github.com/NVIDIA/nccl/releases

Reply:
Thanks for your comments. I changed NCCL 1.0 and NCCL 2.0 to NCCL 1 and NCCL 2, respectively.